python/ray/utils.py

```diff
@@ -780,3 +781,10 @@ def try_to_symlink(symlink_path, target_path):
         os.symlink(target_path, symlink_path)
     except OSError:
         return
+
+
+def get_user():
+    try:
+        return pwd.getpwuid(os.getuid())[0]
```

Comment:
Seems like `pwd.getpwuid` returns named tuple (https://www.geeksforgeeks.org/pwd-module-in-python/)? Can you just use the name instead of index?

Reply:
Done!